src/main/java/io/odpf/depot/common/Template.java

```diff
@@ -50,9 +52,4 @@ private Object fetchInternalValue(Object ob) {
             return ob;
         }
     }
-
-    @Override
-    public String toString() {
```

Comment:
why did we remove this? We can use this in inside to getter.
maybe we can change the name:
    public String getTemplateString() {
        return templatePattern;
    }